### PR TITLE
Updated metatags test and added a wait.

### PR DESCRIPTION
**Pull request by gargsuchi**

### CODE REVIEW

**Review by sonnykt — APPROVED** [review]

File: tests/behat/bootstrap/MiscContext.behat.inc [hunk 1]
```diff
@@ -159,4 +159,28 @@ class MiscContext extends DrupalSubContextBase {
...
+  }
+
+  /**
+   * Click any element.
+   *
+   * @Given I click the :selector element
+   *
+   * @see https://stackoverflow.com/a/33672497/1023773
+   */
+  public function iClickTheElement($selector) {
```
Comment (pandaskii) [inline tests/behat/bootstrap/MiscContext.behat.inc]: What is the usage of this snippet?

File: tests/behat/bootstrap/MiscContext.behat.inc [hunk 2]
```diff
@@ -159,4 +159,28 @@ class MiscContext extends DrupalSubContextBase {
     }
   }
 
+  /**
+   * @Given I wait :arg1 seconds
+   */
+  public function iWaitSeconds($arg1) {
+    sleep($arg1);
```
Comment (pandaskii) [inline tests/behat/bootstrap/MiscContext.behat.inc]: Not sure something like $result = $this->getSession()->wait(5000, $condition); can do the job as well?

**Review by pandaskii — APPROVED** [review]
Looks good, happy to merge after testing.

**Comment by pandaskii** [comment]
CircleCI passed, thanks

**Pull request merged**